app/models/custom/setting.rb

```diff
@@ -24,6 +24,35 @@ def defaults
       else
         consul_defaults.merge({
           # Overwrite default CONSUL settings or add new settings here
+          "homepage.widgets.feeds.debates": false,
+          "homepage.widgets.feeds.processes": false,
+          "homepage.widgets.feeds.proposals": false,
+          "feature.help_page": false,
+          "feature.facebook_login": false,
+          "feature.google_login": false,
+          "feature.twitter_login": false,
+          "feature.wordpress_login": false,
+          "feature.signature_sheets": false,
+          "feature.user.recommendations": false,
+          "feature.graphql_api": false,
+          "feature.sdg": false,
+          "process.debates": false,
+          "process.proposals": false,
+          "process.budgets": false,
+          "process.legislation": false,
+          facebook_handle: nil,
+          instagram_handle: nil,
+          telegram_handle: nil,
+          twitter_handle: nil,
+          twitter_hashtag: nil,
+          youtube_handle: nil,
+          url: "http://example.com",
+          org_name: "Innoviris",
+          meta_title: "Definition pending",
+          meta_description: "Definition pending",
+          meta_keywords: "Definition pending",
+          mailer_from_name: "CONSUL",
+          mailer_from_address: "noreply@consul.dev"
```

Comment:
I'm fine with either alphabetic order or using the same order as used in the original `setting.rb` model. Both would make it easier to see what we've changed.

Maybe I slightly prefer using the same order as in the `setting.rb` model so we can easily see which ones have changed and which ones haven't, but I might change my mind, so if you prefer alphabetic order, that's fine with me :wink:.

Reply:
Done 👍🏼